meta/autoid/memid_test.go

```diff
@@ -16,22 +16,23 @@ package autoid_test
 import (
 	"context"
 	"math"
+	"testing"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/parser/model"
 	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/parser/terror"
-	"github.com/pingcap/parser/types"
 	"github.com/pingcap/tidb/meta/autoid"
 	"github.com/pingcap/tidb/store/mockstore"
+	"github.com/pingcap/tidb/types"
```

Comment:
why change this import?

Reply:
This may related to vscode package matching, I will fix it.